src/Microsoft.OData.Core/Json/JsonWriterAsync.cs

```diff
@@ -245,6 +248,124 @@ public async Task WriteValueAsync(byte[] value)
             await this.writer.WriteValueAsync(value, this.wrappedBuffer, this.ArrayPool).ConfigureAwait(false);
         }
 
+#if NETCOREAPP3_1_OR_GREATER
+        public Task WriteValueAsync(JsonElement value)
+        {
+            switch (value.ValueKind)
+            {
+                case JsonValueKind.Null:
+                    return this.WriteValueAsync((string)null);

```

Comment:
will this be converted to an empty string? `string(null)` why can't you pass directly? or the string equivalent of what `string(null)` gives?

Reply:
I can't pass `WriteValueAsync(null)` because the compiler won't be sure whether to call `WriteValueAsync(string)` or `WriteValueAsync(byte[])`. Since both `string` and `byte[]` can accept `null` as a value, I need to be explicit about wich overload I am targeting otherwise the code won't compile.